homeassistant/components/zha/core/channels/base.py

```diff
@@ -43,6 +44,13 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class ReportConfig(TypedDict, total=True):
+    """Matcher for the dhcp integration for required fields."""
+
+    attr: str
```

Comment:
In the reporting configuration dict, attribute could be either an attribute name `str` or an attribute Id - `int`